sys/include/net/gnrc/netdev2.h

```diff
@@ -81,8 +96,46 @@ typedef struct gnrc_netdev2 {
      * @brief PID of this adapter for netapi messages
      */
     kernel_pid_t pid;
+
+#ifdef MODULE_GNRC_MAC
+    uint16_t mac_info;  /**< general information for the MAC protocol */
+#endif
 } gnrc_netdev2_t;
 
+#ifdef MODULE_GNRC_MAC
```

Comment:
@miri64 I also `#ifdef MODULE_GNRC_MAC` the `gnrc_mac` related functions, is it OK?